site/content/en/_index.html

```diff
@@ -0,0 +1,42 @@
++++
+title = "Kueue"
+linkTitle = "Kueue"
+
++++
+
+{{< blocks/cover title="Welcome to Kueue" image_anchor="top" height="full" color="orange" >}}
+<div class="mx-auto">
+	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
+		Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
```

Comment:
```suggestion
		Read the documentation <i class="fas fa-arrow-alt-circle-right ml-2"></i>
```
Or just `Documentation`?